src/App.tsx

```diff
@@ -1,9 +1,34 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
+import { Movies } from './types/movies';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const handleChangeFilterText = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setQuery(e.target.value);
+  };
+
+  const visibleMovies = (moviesArray: Movies[], filter: string): Movies[] => {
+    if (filter.length) {
+      return moviesArray.filter(el => {
+        const { description, title } = el;
+        const descriptionFilter = description
+          .toLocaleLowerCase()
+          .includes(filter.toLocaleLowerCase().trim());
+        const titleFilter = title
+          .toLocaleLowerCase()
+          .includes(filter.toLocaleLowerCase().trim());
```

Comment:
[checklist](https://github.com/mate-academy/react_movies-list-filter/blob/master/checklist.md): Don't repeat yourself. If you see that you copy/paste some block code, maybe, it's time to create a separate variable/function